target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -47,6 +47,10 @@ case "$FIRMWARE" in
 		caldata_extract_mmc "0:ART" 0x9000 0x2f20
 		ath10k_patch_mac $(mmc_get_mac_binary ARTMTD 0x12)
 		;;
+	 netgear,ex6200v2)
+                caldata_extract_mmc "0:ART" 0x170000 0x2f20
```

Comment:
Why do you need hotplug script for caldata if its already set via NVMEM?

Reply:
Following what done for commits of adding support to similar devices, namely netgear ORBI. Sounds like I can remove this from this file and still have WIFI work?